packages/swatch/test/swatch-group.test.ts

```diff
@@ -365,10 +369,13 @@ describe('Swatch Group - DOM selected', () => {
         `);
 
         await elementUpdated(el);
+        await nextFrame();
+        await nextFrame();
 
         expect(el.selected).to.deep.equal(['color-1', 'color-3']);
     });
-    it('lazily accepts selection from DOM', async () => {
+    it('lazily accepts selection from DOM', async function () {
+        this.retries(0);
```

Comment:
I think you could set `this.retries(0)` at the `describe` level, if you wanted to for this block of tests.